.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,39 @@
+---
+
+name: Bug Report
+about: Create a bug report to help us improve pandas
+title: "BUG:"
+labels: ""
+
+---
+
+- [ ] I have checked that issue has not already been reported.
+
+- [ ] I have confirmed this bug exists on the latest version of pandas.
+
+- [ ] I have confirmed this bug exists on the master branch of pandas.
```

Comment:
I don't think we should necessarily expect that users have master built to report a bug

Reply:
Agreed. Perhaps something like

```suggestion
- [ ] (optinal) I have confirmed this bug exists on the master branch of pandas.
```

which makes the others sound more required too.